libraries/entities-renderer/src/textured_particle.slf

```diff
@@ -10,13 +10,27 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
 
+<@if not HIFI_USE_FORWARD@>
+    <@include DeferredBufferWrite.slh@>
+    <@include CullFace.slh@>
+
+    layout(location=RENDER_UTILS_ATTR_NORMAL_WS) in vec3 _normalWS;
+    layout(location=RENDER_UTILS_ATTR_PREV_POSITION_CS) in vec4 _prevPositionCS;
+<@else@>
+    layout(location=0) out vec4 _fragColor0;
+<@endif@>
+
 LAYOUT(binding=0) uniform sampler2D colorMap;
 
 layout(location=0) in vec4 varColor;
 layout(location=1) in vec2 varTexcoord;
 
-layout(location=0) out vec4 outFragColor;
-
 void main(void) {
-    outFragColor = texture(colorMap, varTexcoord.xy) * varColor;
```

Comment:
this is the crux of the fix for the AMD bug, which is also applied to all the other shaders I could find that weren't doing it properly.

basically, on AMD cards, if you have an FBO bound with multiple attachments, but only output to one of them, they will all get borked.  the fix is to make sure that when we're using forward rendering, we only output to location 0, but otherwise (including for transparent objects, which were causing the problem), we are outputting to all of the attachments via the packDeferredFragmentXXX methods